sdks/wasm/src/driver.c

```diff
@@ -280,8 +279,10 @@ icall_table_lookup (MonoMethod *method, char *classname, char *methodname, char
 	void *p = bsearch (&token_idx, indexes, indexes_size, 4, compare_int);
 	if (!p) {
 		return NULL;
+		/*
```

Comment:
is this intentional?

Reply:
Its unreachable code.

Reply:
ok, i just meant 'is it intentional for it to be a comment instead of deleted'

Reply:
Its debug code which can be enabled.